transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueDomainSocketChannel.java

```diff
@@ -41,6 +41,10 @@ public KQueueDomainSocketChannel() {
         super(null, newSocketDomain(), false);
     }
 
+    public KQueueDomainSocketChannel(int fd) {
+        super(new BsdSocket(fd));
```

Comment:
for `epoll` this sets active to `true` but this will set active depending on the `isSoErroZero` method. Consider just calling `this(null, new BsdSocket(fd));`